content/docs/latest/nebraska/authorization/_index.md

```diff
@@ -47,101 +47,153 @@ because the defaults are set at half.
+    quay.io/keycloak/keycloak:26.3.4 \
+    start-dev
+  ```
 
-- Start the Nebraska backend:
-
-  - `nebraska --auth-mode oidc --oidc-admin-roles nebraska_admin  --oidc-viewer-roles nebraska_member --oidc-client-id nebraska --oidc-issuer-url http://localhost:8080/auth/realms/master --oidc-client-secret <Your_Client_Secret>`
-
-  Note: If roles array in the token is not in `roles` key, one can specify a custom JSON path using the `oidc-roles-path` flag.
-
-- In the browser, access `http://localhost:8000`
+- Open http://localhost:8080 in your browser to access keycloak UI and login with the username admin and password as admin.
 
-# Preparing Keycloak as an OIDC provider for Nebraska
+## Creating a client
 
-- Run `Keycloak` using docker:
-    - `docker run -p 8080:8080 -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=admin -d quay.io/keycloak/keycloak:13.0.1`
+1. Click on `Clients` menu option and click `Create`.
+2. Set the client name as `nebraska` and click `Save`.
+3. Change the `Access Type` to `public` (for SPA/public client).
+4. Ensure `Direct Access Grants Enabled` is OFF (not needed for PKCE flow).
+5. Ensure `Standard Flow Enabled` is ON (for Authorization Code Flow).
+6. Set `Valid Redirect URIs` to `http://localhost:8000/auth/callback`.
+7. Set `Valid post logout redirect URIs` to `http://localhost:8000/`.
+8. Set `Web Origins` to `http://localhost:8000` (use `+` to allow all Valid Redirect URIs origins for CORS).
 
-- Open http://localhost:8080 in your browser to access keycloak UI and login with the username admin and password as admin.
+{{< presentation "keycloak-create-client" >}}
 
 ## Creating Roles
 
 ### Member Role
 
-1. Click on `Roles` menu option and select `Add Role`.
+1. Click on `Roles` menu option and select `Create Role`.
 2. Provide a name for the member role, here we will use `nebraska_member`.
 3. Click `Save`.
 
 ### Admin Role
 
-1. Click on `Roles` menu option and select `Add Role`.
+1. Click on `Roles` menu option and select `Create Role`.
 2. Provide a name for the admin role, here we will use `nebraska_admin`.
 3. Click `Save`.
-4. After the admin role is created enable composite role to ON. In the Composite Roles section select the member role, In our case it is nebraska_member and click Add Selected.
+4. After the admin role is created enable composion: Go to associated roles, and add `nebraska_member`.
 
 Now the member and admin roles are created, the admin role is a composite role which comprises of member role.
 
-<p align="center">
-  <img width="100%"  src="../images/keycloak-roles.gif">
-</p>
+{{< presentation "keycloak-create-roles" >}}
 
-## Creating a client
+## Adding roles scope to token
 
-1. Click on `Clients` menu option and click `Create`.
-2. Set the client name as `nebraska` and click `Save`.
-3. Change the `Access Type` to `Confidential`
-4. Set `Valid Redirect URIs` to `http://localhost:8000/login/cb`.
+1. Click on `Client Scopes > nebraska-dedicated`.
```

Comment:
In "nebraska" client details screen, find "Client scopes" tab and select "nebraska-dedicated" scope.